sql/parser/alter.go

```diff
@@ -5,40 +5,67 @@ import (
 	"github.com/genjidb/genji/sql/scanner"
 )
 
+func (p *Parser) parseAlterTableRenameStatement(tableName string) (_ query.AlterStmt, err error) {
+	var stmt query.AlterStmt
+	stmt.TableName = tableName
+
+	// Parse "TO".
+	if tok, pos, lit := p.ScanIgnoreWhitespace(); tok != scanner.TO {
+		return stmt, newParseError(scanner.Tokstr(tok, lit), []string{"TO"}, pos)
+	}
+
+	// Parse new table name.
+	stmt.NewTableName, err = p.parseIdent()
+	if err != nil {
+		return stmt, err
+	}
+
+	return stmt, nil
+}
+
+func (p *Parser) parseAlterTableAddFieldStatement(tableName string) (_ query.AlterTableAddColumn, err error) {
+	var stmt query.AlterTableAddColumn
+	stmt.TableName = tableName
+
+	// Parse "COLUMN".
+	if tok, pos, lit := p.ScanIgnoreWhitespace(); tok != scanner.COLUMN {
+		return stmt, newParseError(scanner.Tokstr(tok, lit), []string{"COLUMN"}, pos)
+	}
```

Comment:
>Original issue requests ADD FIELD statement, but I think ADD COLUMN is more compatible with SQL. Anyway, it's not too hard to fix.

Perhaps we can allow both keywords and use the AddField in internal API (`s/AddColumn/AddField`)?